main.js

```diff
@@ -85,76 +88,83 @@ function triggerAlert(alertNeeded) {
 function switchView() {
   var goalForm = document.querySelector(".main-panel");
   var timerView = document.querySelector(".timer-view");
-  if (currentActivity.category === "study") {
+  if (currentActivity.category === "Study") {
     document.querySelector(".timer-btn").classList.add("study-ring")
   }
-  if (currentActivity.category === "meditate") {
+  if (currentActivity.category === "Meditate") {
     document.querySelector(".timer-btn").classList.add("meditate-ring")
   }
-  if (currentActivity.category === "exercise") {
+  if (currentActivity.category === "Exercise") {
     document.querySelector(".timer-btn").classList.add("exercise-ring")
   }
   goalForm.classList.toggle("hidden");
   timerView.classList.toggle("hidden");
-  formatTime();
+  var currentSeconds = (currentActivity.minutes * 60) + currentActivity.seconds;
+  formatRemainingTime(currentSeconds);
 }
 
 function displayInput(time) {
   document.querySelector(".goal-phrase").innerText = `${currentActivity.description}`;
   document.querySelector(".goal-duration").innerText = `${time}`;
 }
-
-function formatTime() {
-  var seconds = currentActivity.seconds;
-  var minutes = currentActivity.minutes;
-  if (currentActivity.seconds < 10) {
-    seconds = `0${currentActivity.seconds}`;
-  }
-  if (currentActivity.minutes < 10) {
-    minutes = `0${currentActivity.minutes}`;
-  }
-  displayInput(`${minutes}:${seconds}`);
-}
+//
+// function formatTime() {
+//   var seconds = currentActivity.seconds;
+//   var minutes = currentActivity.minutes;
+//   if (currentActivity.seconds < 10) {
+//     seconds = `0${currentActivity.seconds}`;
+//   }
+//   if (currentActivity.minutes < 10) {
+//     minutes = `0${currentActivity.minutes}`;
+//   }
+//   displayInput(`${minutes}:${seconds}`);
+// }
 
 function formatRemainingTime(time) {
-  currentActivity.minutes = Math.floor(time / 60);
-  currentActivity.seconds = time % 60;
-  var seconds = currentActivity.seconds;
-  var minutes = currentActivity.minutes;
+  var minutes = Math.floor(time / 60);
+  var seconds = time % 60;
 
-  if (currentActivity.seconds < 10 && currentActivity.seconds > 0) {
-    seconds = `0${currentActivity.seconds}`;
-  }
-  if (currentActivity.seconds < 1) {
-    seconds = '00';
-  }
-  if (currentActivity.minutes < 10 && currentActivity.minutes > 0) {
-    minutes = `0${currentActivity.minutes}`;
-  }
-  if (currentActivity.minutes === 0) {
-    minutes = '00'
-  }
+  if (seconds < 10 && seconds > 0) {seconds = `0${seconds}`;}
+  if (seconds < 1) {seconds = '00';}
+  if (minutes < 10 && minutes > 0) {minutes = `0${minutes}`;}
+  if (minutes === 0) {minutes = '00'}
```

Comment:
Multiple if statements in a row is usually a good place to start refactoring. Maybe something like [padStart()]( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) would be a bit cleaner? (caveat: I have not actually used this, I just found it now)